cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -211,8 +213,13 @@ protected static <C extends KafkaConnectCluster> C fromSpec(Reconciliation recon
             kafkaConnect.setConfiguration(config);
         }
         if (kafkaConnect.tracing != null)   {
-            config.setConfigOption("consumer.interceptor.classes", "io.opentracing.contrib.kafka.TracingConsumerInterceptor");
-            config.setConfigOption("producer.interceptor.classes", "io.opentracing.contrib.kafka.TracingProducerInterceptor");
+            if (JaegerTracing.TYPE_JAEGER.equals(kafkaConnect.tracing.getType())) {
+                config.setConfigOption("consumer.interceptor.classes", "io.opentracing.contrib.kafka.TracingConsumerInterceptor");
+                config.setConfigOption("producer.interceptor.classes", "io.opentracing.contrib.kafka.TracingProducerInterceptor");
```

Comment:
Can we have some constants for those classes (config options)?

Reply:
I don't see big value on using constants here. The Jaeger part will go away at some point soon anyway.

Reply:
My point was that when you'll have for example one class which would hold these information for the Jaeger part, once we'll remove it, it will be much easier and we will be sure that we didn't forget anything 🤷 . But yeah, it was just an idea.

Reply:
I can create constants for these classes and the corresponding ones for OpenTelemetry. The open question is where?
These classes are used mostly in the model and assembly related stuff. We could think of using the `ModelUtils` as a central place or having them in the `api` module in each `xxxxTracing` class definition but it would be exposed to users who use our api. I have no strong opinion. @scholzj what's your take on this?

Reply:
Ok, `api` module works for me. I just noticed the usage with Authorization classes, so it makes sense doing the same way.